src/settings-ui/Settings.UI/OOBE/Views/OobeRegistryPreview.xaml.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.PowerToys.Settings.UI.Library;
+using Microsoft.PowerToys.Settings.UI.OOBE.Enums;
+using Microsoft.PowerToys.Settings.UI.OOBE.ViewModel;
+using Microsoft.PowerToys.Settings.UI.Views;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Navigation;
+
+namespace Microsoft.PowerToys.Settings.UI.OOBE.Views
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class OobeRegistryPreview : Page
+    {
+        public OobePowerToysModule ViewModel { get; set; }
+
+        public OobeRegistryPreview()
+        {
+            this.InitializeComponent();
+            ViewModel = new OobePowerToysModule(OobeShellPage.OobeShellHandler.Modules[(int)PowerToysModules.RegistryPreview]);
+            DataContext = ViewModel;
+        }
+
+        private void Launch_RegistryPreview_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+        {
+            ShellPage.SendDefaultIPCMessage("{\"action\":{\"RegistryPreview\":{\"action_name\":\"Launch\", \"value\":\"\"}}}");
+        }
+
+        private void SettingsLaunchButton_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
```

Comment:
Launch button launches the app even if Registry Preview is disabled in settings. This should not be the case.

We can allso add launch button to the settings page. Not a blocker, though. Can be a follow-up PR.

Reply:
Interesting.  I think someone with more experience in the launcher should get involved in this one: I put in the enable/disable code based off the examples of other apps but could have missed this.

Reply:
Perfect that's great, please kindly summarize with a single click url login